internal/ifelse/branch_kind.go

```diff
@@ -58,19 +58,19 @@ func (k BranchKind) String() string {
 	case Empty:
 		return ""
 	case Regular:
-		return "..."
+		return ""
```

Comment:
Do we need to change this?

Reply:
Yeah, because this stringer participates in the new linter message where ellipses are not needed, adding them in now happens at a higher level.

Reply:
Ok, you may resolve this comment.